doc/api/crypto.md

```diff
@@ -1196,7 +1196,8 @@ rapidly.
 In line with OpenSSL's recommendation to use pbkdf2 instead of
 [`EVP_BytesToKey`][] it is recommended that developers derive a key and IV on
 their own using [`crypto.pbkdf2()`][] and to use [`crypto.createCipheriv()`][]
-to create the `Cipher` object.
+to create the `Cipher` object. A warning is emitted when counter mode (e.g. CTR,
+GCM or CCM) is used in `crypto.createCipher()` in order to avoid IV reuse.
```

Comment:
I changed the doc into a little stronger description as ae7dbb1ca5144df33115542e0baf4cf9e2ee1ed3 and add a reference of Nonce-Disrespecting Adversaries. 

> Would it be worthwhile to mention that this API predates the addition of counter-based ciphers to OpenSSL, hence why it's not an error outright?

I think it gets more complicated description in the doc. We can make throwing an error some time later after enough time.